### PR TITLE
Change Prometheus cluster name metrics label to verrazzano_cluster for v1.1

### PR DESCRIPTION
**Pull request by desagar**
…or v1.1

### CODE REVIEW

File: content/en/docs/setup/install/multicluster.md [hunk 1]
```diff
@@ -285,7 +285,8 @@ You can perform all the verification steps on the admin cluster.
 ### Verify that managed cluster metrics are being collected
 
 Verify that the admin cluster is collecting metrics from the managed cluster.  The Prometheus output will include 
-records that contain the name of the managed cluster (labeled as `managed_cluster`).
+records that contain the name of the managed cluster, labeled as `verrazzano_cluster`, in addition to admin cluster
+records which are labeled as `local`
```
Comment (mgianatagh) [inline content/en/docs/setup/install/multicluster.md:289]: This sentence seems to convey there is a label named 'local' on admin clusters, instead a a label named verrazzano_cluster with a value of local.

Comment (desagar) [reply content/en/docs/setup/install/multicluster.md]: updated the text

**Review by mgianatagh — APPROVED** [review]

File: content/en/docs/setup/install/multicluster.md [hunk 2]
```diff
@@ -285,7 +285,8 @@ You can perform all the verification steps on the admin cluster.
 ### Verify that managed cluster metrics are being collected
 
 Verify that the admin cluster is collecting metrics from the managed cluster.  The Prometheus output will include 
-records that contain the name of the managed cluster (labeled as `managed_cluster`).
+records that contain the name of the managed cluster, labeled as `verrazzano_cluster`, in addition to admin cluster
+records which contain the value `local` for the `verrazzano_cluster` label.
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/multicluster.md:289]: records which contain -> records, (comma) which contain

**Review by rosemarymarano — APPROVED** [review]
One very minor edit.

**Pull request closed without merging**